python-template/{{cookiecutter.placeholder_repo_name}}/.travis.yml

```diff
@@ -0,0 +1,48 @@
+# Config file for automatic testing at travis-ci.org
+# To connect a new edX repository to Travis, see
+# https://openedx.atlassian.net/wiki/spaces/EdxOps/pages/157464369/How+to+connect+a+repository+to+Travis+CI
+#
+# Be sure to change the Travis settings in the web UI to enable both
+# "Build pushed branches" and "Build pushed pull requests".
+
+language: python
+
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+  - 3.8
+
+matrix:
+  include:
+    - python: 3.6
+      env: TOXENV=quality
+    - python: 3.6
+      env: TOXENV=docs
+    - python: 3.6
+      env: TOXENV=pii_check
```

Comment:
I think this should be 3.5 and 3.8, and not 3.6?  See the Python 3.8 tasks for details.